modules/tv/search.php

```diff
@@ -281,7 +281,7 @@
             }
         // If the next thing starts with stars, it's a movie rating query
             if (preg_match('#(\\*+\s*(1/2\b|0?\.5\b|-)?)\s*(.*?)$#', $search_str, $match)) {
-                $starcount = substr_count($match[1], '*') / 4;
+                $starcount = substr_count($match[1], '*') / 5;
```

Comment:
Is this a magic value for max_stars? Defaults to 4, you suggest changing it to 5?